src/pages/Feeds/components/FeedListView.tsx

```diff
@@ -145,40 +148,33 @@ const FeedListView: React.FC = () => {
       </React.Fragment>
     );
   }
+
+  const style = { fontSize: "1.15em" };
+
   return (
     <React.Fragment>
-      <PageSection variant={PageSectionVariants.light} className="feed-header">
-        <div className="feed-header__split">
-          <Title headingLevel="h1" size="3xl">
-            New and Existing Analyses
-            {totalFeedsCount > 0 ? (
-              <span className="feed-header__count">({totalFeedsCount})</span>
-            ) : null}
-          </Title>
-          <CreateFeedProvider>
-            <CreateFeed />
-          </CreateFeedProvider>
-        </div>
-
-        <Hint
-          //@ts-ignore
-          style={{
-            width: `${width > 768 ? "50%" : "100%"}`,
-            // paddingBottom: '0',
-            marginTop: "20px",
-          }}
-        >
-          <HintBody
-            //@ts-ignore
-            style={{
-              gridColumn: `${width > 768 ? "1" : "2"}`,
-            }}
-          >
-            All Analyses that you have completed are recorded here. You can
-            easily return to a completed analysis and add more analysis
-            components, or you can create a brand new analysis from scratch.
-          </HintBody>
-        </Hint>
+      <PageSection className="feed-header" variant="light">
+        <InfoIcon
+          title={`New and Existing Analyses
+            ${
+              totalFeedsCount > 0 ? (
+                <span className="feed-header__count">({totalFeedsCount})</span>
+              ) : (
+                "(0)"
+              )
+            }`}
+          p1={
+            <Paragraph style={style}>
+              All Analyses that you have completed are recorded here. You can
```

Comment:
It is a good idea to have a tooltip on this page. May I suggest some verbage? @rudolphpienaar and @mairin please comment.

> Analyses (aka _ChRIS_ feeds) are computational experiments where data are organized and processed by ChRIS plugins. In this view you may view your analyses and also the ones shared with you.

Reply:
Hi @Timonwa  If you're still around, could you please change the verbage to what Jennings has proposed. This PR is good to be merged after. Please merge the latest changes in your master to avoid merge conflicts. Sorry for the delay on this.

Reply:
ok. no problem.